lib/crypto/src/crypto.erl

```diff
@@ -779,21 +843,18 @@ enable_fips_mode_nif(_) -> ?nif_stub.
 equal_const_time(X1, X2) ->
     equal_const_time(X1, X2, true).
 
-
-equal_const_time(<<B1,R1/binary>>, <<B2,R2/binary>>, Truth) ->
+equal_const_time(<<B1, R1/binary>>, <<B2, R2/binary>>, Truth) ->
```

Comment:
Another interesting case :) What would you propose as a consistent way to format binary pattern matching? Sorry for asking, but I struggle to think about how to solve this and you seem to have new ideas.

Reply:
I'm just happy to tell what I like and dislike :)
In arguments or elements I personally prefer not to have spaces in the binary as in the commented line, but as standalone patterns or topmost right expressions it could be clearer:
`<<Code, Rest/binary>> = Data,`
 and
`NewData = <<Code, SomethingElse, Rest/binary>> ,`

Reply:
Ha ha, yes even knowing what you like/dislike is also helpful :)

Okay so in this case it would also be tough to make an automated rule, or am I misinterpreting?